tools/build_variables.bzl

```diff
@@ -42,6 +42,7 @@ def libtorch_generated_sources(gencode_pattern):
     ]] + [
         "torch/csrc/autograd/TraceTypeManual.cpp",
         "torch/csrc/autograd/VariableTypeManual.cpp",
+        "torch/csrc/autograd/FunctionsManual.cpp",
```

Comment:
I'm confused because none of these three files are autogenerated. But as long as the build work this seems fine

Reply:
They correspond to the functions that are missing from the codegen. So they are at the same place.